patient-appointment-management/.env

```diff
@@ -0,0 +1,88 @@
+# description: Customer name to be displayed in SMS.
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_NAME=
+
+# description: Customer short name. Must be all lowercase one word without any spaces.
+# format: text
+# required: true
+# configurable: true
+APPLICATION_CUSTOMER_CODE=
+
+# description: Customer EHR endpoint url for communication from Twilio via HTTP POST
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_EHR_ENDPOINT_URL=
+
+# description: Twilio phone number to send SMS from. Your patient will see this number.
+# format: phone_number
+# required: true
+# configurable: true
+TWILIO_PHONE_NUMBER=
+
+# description: Reminder outreach window start time local timezone (HHMM) inclusive. default 0000
```

Comment:
Any link available explaining the timezone format specifically, for those who haven't handled numeric timezones before? Also, what does start time mean in this context?

Reply:
I'll re-write the description to be more clear. It's not the time zone format, but the HHMM (in local time zone) at which the reminders should be allowed to be sent out. So, 0900 just implies 9am in whatever the local time zone is. makes sense?

Reply:
That makes sense. I assume the timezone is calculated from the Lambda's configured region?

Reply:
Correct, because lambda will run in GMT, so the separate timezone variable will be used to convert the local timezone appointment to GMT. Javascript/nodejs wasn't able to parse the timezone correctedly from ISO 8601 string... hence this workaround.